default.nix

```diff
@@ -4,6 +4,7 @@ in
 { system ? builtins.currentSystem
 , config ? {}
 , dconfig ? "testnet_staging"
+, gitrev ? "unknown"
```

Comment:
https://github.com/NixOS/hydra/blob/7e9b6acbce9e1faf25c1f43e441490a5167174af/release.nix#L1

https://hydra.iohk.io/jobset/serokell/cardano-sl#tabs-configuration

an attrset called `cardano` will be passed into `release.nix`, containing `outPath`, `revCount`, and `rev`, `release.nix` then needs to pass that to `default.nix`